docs/rules/indent.md

```diff
@@ -87,6 +88,9 @@ Level of indentation denotes the multiple of the indent specified. Example:
 * Indent of 2 spaces with `MemberExpression` set to `0` will indent the multi-line property chains with 0 spaces.
 * Indent of 2 spaces with `MemberExpression` set to `1` will indent the multi-line property chains with 2 spaces.
 * Indent of 2 spaces with `MemberExpression` set to `2` will indent the multi-line property chains with 4 spaces.
+* Indent of 2 spaces with `BinaryExpression` set to `0` will indent the multi-line continuations with 0 spaces.
```

Comment:
What would be the use-case for indenting with user-defined number of indentation levels? Shouldn't we always indent by just 1 indentation level?